bitnami/external-dns/README.md

```diff
@@ -170,7 +170,8 @@ The following table lists the configurable parameters of the external-dns chart
 | `service.loadBalancerSourceRanges`  | List of IP CIDRs allowed access to load balancer (if supported)                                          | `[]`                                                        |
 | `service.annotations`               | Annotations to add to service                                                                            | `{}`                                                        |
 | `rbac.create`                       | Weather to create & use RBAC resources or not                                                            | `true`                                                      |
-| `rbac.serviceAccountName`           | ServiceAccount (ignored if rbac.create == true)                                                          | `default`                                                   |
+| `rbac.serviceAccount.create`        | If true and rbac.create is also true, a service account will be created                                  | `true`                                                      |
```

Comment:
I still think this parameter is useless, I mean we can obtain the same behaviour without it. See my other comments

Reply:
Thank you for your polite review !

Sorry for not being able to communicate well in English...

`rbac.serviceAccount.create` parameter cannot be removed. 
The intent of this PR is to create status in which `clusterrole (role)` and `clusterrolebiding (rolebinding)` are created but `serviceAccount` is not created.

rolebinding is setting for the serviceAccount determined in https://github.com/bitnami/charts/pull/2088/files#diff-3d3105b5ff96a11099b64bae938ea60dR431-R441.

Because, when using iam role for sa (IRSA) in EKS, the simplest method is to create a serviceAccount that supports IRSA with `eksctl`. And by using serviceAccount created by eksctl, application can use iam role for sa.

FYI: https://aws.amazon.com/jp/blogs/opensource/introducing-fine-grained-iam-roles-service-accounts/

Reply:
I see your point, but you can get the same without that parameter (check my comment at `_helpers.tpl`)
This is your current logic:
- `rbac.create=true` && `rbac.serviceAccount.create` -> Create SA using `{{ include "external-dns.fullname" . }}`
- `rbac.create=true` && `rbac.serviceAccount.create=false` -> Reuse existing SA (name taken from `rbac.serviceAccountName`)
This is what I suggest:
- `rbac.create=true` && `rbac.serviceAccountName is NOT defined` -> Create SA using `{{ include "external-dns.fullname" . }}`
- `rbac.create=true` && `rbac.serviceAccountName is defined` Reuse existing SA (name taken from `rbac.serviceAccountName`)
As you can see, the result is the same but you're just using one parameter: `rbac.serviceAccountName`

Reply:
what does it mean for https://github.com/bitnami/charts/pull/2088#discussion_r395470884 to be a revert?

If I revert this, the `rbac.serviceAccount.create` parameter will not work in `serviceaccount.yaml`, and the `rbac.create` parameter alone will determine whether the sa is created or not.

I'm sorry if I'm mistaken..